src/views/ThreadPage/Comment/CreateCommentForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, ChangeEvent, FormEvent } from 'react';
+import { useParams } from 'react-router-dom';
+import useAxiosFunction from '../../../hooks/useAxiosFunction';
+import './CommentForm.css';
+
+interface CommentData {
+  author_id: number;
+  content: string;
+}
+
+interface CommentFormProps {
+  refetchComments: () => void;
+}
+
+const CreateCommentForm: React.FC<CommentFormProps> = ({ refetchComments }) => {
+  const { id } = useParams<{ id: string }>();
+
+  const [commentData, setCommentData] = useState<CommentData>({
+    author_id: 1, // TODO: we can remove this once we have auth
+    content: '',
+  });
+
+  const [response, error, loading, axiosFetch] = useAxiosFunction();
+
+  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    const { name, value } = event.target;
+    setCommentData({ ...commentData, [name]: value });
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    axiosFetch({
+      method: 'POST',
+      url: `/api/v1/threads/${id}/comments`,
+      data: {
+        ...commentData,
+        author_id: 1, // TODO remove this once we have auth
+      },
+    }).then(() => {
+      setCommentData({
+        ...commentData,
+        content: '',
+      });
+      refetchComments();
```

Comment:
In this form, along with the edit and reply forms, I've decided to just use the re-fetch strategy here because of all the nesting. Is refetching alright here, or do you think I should update the local state variable instead?

Reply:
Actually come to think of it updating the state locally would be better for performance (so we don't have to re-fetch and re-render all comment trees). I could change that for this PR.

Reply:
Actually on third thought, it might be kinda tough to do right now. Do you want to make that a separate ticket (and potentially tackle it after we decide on a state management system)?

Reply:
Yea re-fetching is a good move for now, there are many components that the comment is nested under so it would be a pain to pass the props down (also recursively). I think we should move towards using a state management library in the future. We should probably make a new ticket for that migration. Also, just some food for thought: what would happen if two people comment under the same comment at the same time. Do we want to refetch and get the comments after submission?This way both replies will display after submission. Otherwise, if we do it by state, when we submit new comments will not be displayed unless the page is reloaded.